python/cuml/dask/naive_bayes/naive_bayes.py

```diff
@@ -242,25 +242,24 @@ def predict(self, X):
         x_worker_parts = workers_to_parts(gpu_futures)
 
         key = uuid1()
-
         futures = [(wf[0],
                     self.client_.submit(MultinomialNB._get_size,
                                         wf[1],
                                         workers=[wf[0]],
                                         key="%s-%s" % (key, idx)))
                    for idx, wf in enumerate(gpu_futures)]
 
-        sizes = self.client_.compute(list(map(lambda x: x[1],
-                                              futures)), sync=True)
+        sizes = self.client_.compute([x[1] for x in futures], sync=True)
 
-        models = dict([(w, self.client_.scatter(self.model_,
-                                                broadcast=True,
-                                                workers=[w]))
-                       for w, p in x_worker_parts.items()])
+        models = self.client_.scatter(self.model_,
```

Comment:
does this need the traverse=True treatment?

Reply:
I don’t believe so. Scatter doesn’t have a traverse option and the tests are not failing from this code path. It looks like it’s on dask.delayed, client.submit, and client.compute. Hoping @jakirkham might advise.